README.md

```diff
@@ -53,6 +53,10 @@ Plugin tries to load Google Account in the following order:
 - Specified settings in sbt build
 ```
 googleCredentialsFile := Some(new File("<your-account-file>"))
+
+// or
+
+googleCredentialsAccessToken := Some("<your-access-token-here>")
```

Comment:
Q: I wonder if the explicit parameter is more appropriate for temporarily allocated tokens or just some environment variables (from CI)? In case developers locally use gcloud auth or account file this config seems to need to be patched all the time for different envs?

Reply:
Also I don't think having token in plain text here is a good idea. SBT file in general should avoid sensitive information in it, so I incline not to introduce this option but introduce environment variable.

Reply:
Google default authentication is relying on env variables as well, so this is kind of something similar.

Reply:
Would it end up looking like this? if so, any thoughts on the env var name? 

```scala

      .orElse {
        Option( System.getenv( "GAR_ACCESS_TOKEN" ) ).map( accessToken =>
          GoogleCredentials
            .create( AccessToken.newBuilder().setTokenValue( accessToken ).build() )
            .createScoped( scopes )
        )

```

Reply:
Yes, something around it. I found on the Internet people use something like `GOOGLE_OAUTH_ACCESS_TOKEN` :)